src/elements/title/title.visual.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { html } from 'lit';
+
+import { describeViewports, visualDiffDefault } from '../core/testing/private.js';
+
+import '../title.js';
+
+describe(`sbb-title`, () => {
+  describeViewports({ viewports: ['zero', 'small', 'large', 'ultra'] }, () => {
```

Comment:
I'm conflicted about whether we should test all the viewports :/
@jeripeierSBB what do you think?

Reply:
I think the best would be ['zero', 'small', 'medium', 'large', 'ultra'] as for all those values, some title sizes are toggled specifically

Reply:
or maybe just use every viewport size...